itests/kit/deals.go

```diff
@@ -43,6 +43,29 @@ func NewDealHarness(t *testing.T, client api.FullNode, miner TestMiner) *DealHar
 	}
 }
 
+func (dh *DealHarness) MakeFullDealNoRetrieval(ctx context.Context, rseed int, fastRet bool, startEpoch abi.ChainEpoch) ([]byte,
+	*api.DealInfo, cid.Cid) {
+	res, _, data, err := CreateImportFile(ctx, dh.client, rseed, 0)
+	if err != nil {
+		dh.t.Fatal(err)
+	}
+
+	fcid := res.Root
+	fmt.Println("FILE CID: ", fcid)
+
+	deal := dh.StartDeal(ctx, fcid, fastRet, startEpoch)
+
+	// TODO: this sleep is only necessary because deals don't immediately get logged in the dealstore, we should fix this
+	time.Sleep(time.Second)
+	dh.WaitDealSealed(ctx, deal, false, false, nil)
+
+	// Retrieval
+	info, err := dh.client.ClientGetDealInfo(ctx, *deal)
+	require.NoError(dh.t, err)
+
+	return data, info, fcid
+}
+
```

Comment:
This is going to change, but one thing we should try to avoid doing is adding bespoke methods to the test kit that do overly specific things. In this case, an alternative could be to wrap the make `MakeFullDeal` method parameters in a struct, and provide an option "PerformRetrieval" to decide whether we do a retrieval or not, instead of duplicating the code here and removing a few lines.

Reply:
This is done.